test/js-api/jsapi.js

```diff
@@ -16,6 +16,9 @@
 
 const kC0DEFEFE = new Uint8Array([0xC0, 0xDE, 0xFE, 0xFE]);
 
+// Polyfill `globalThis` proposal.
+const globalThis = this;
```

Comment:
Nit: `this.globalThis = this` would be a more accurate polyfill. Top-level `const` doesn't create a global.

Reply:
Thanks, done.